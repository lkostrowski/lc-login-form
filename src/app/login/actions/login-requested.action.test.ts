import { createLoginRequestedAction } from './login-requested.action';

describe('Login Requested Action', function() {
    it('Should match snapshot', () => {
        expect(
            createLoginRequestedAction({
                keepLogged: false,
                email: 'some-email@test.com',
                password: 'Password1',
            }),
        ).toMatchSnapshot();
    });
});
