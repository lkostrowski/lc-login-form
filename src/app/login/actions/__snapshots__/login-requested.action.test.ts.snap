// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Requested Action Should match snapshot 1`] = `
Object {
  "payload": Object {
    "email": "some-email@test.com",
    "keepLogged": false,
    "password": "Password1",
  },
  "type": "login:login-requested",
}
`;
